src/snipe/cli/cli_qc_parallel.py

```
@@ -0,0 +1,617 @@
+import os
+import sys
+import time
+import logging
+from typing import Optional, Any, List, Dict, Set, Union
```

Ruff

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)